smalloc.c

```diff
@@ -12,6 +12,14 @@
 #include <sys/types.h>
 #include <limits.h>
 #include <fcntl.h>
+#include "config-host.h"
```

Comment:
Not needed, config-host.h is included automatically.

Reply:
OK, I will remove that #include directive.